echopop/nwfsc_feat/ingest_sv.py

```diff
@@ -0,0 +1,701 @@
+
+    Examples
+    --------
+    >>> thresholds = {18.0: {"min": -90., "max": -50.},
+    ...               38.0: {"min": -85., "max": -45.}}
+    >>> filtered_data = apply_Sv_thresholds(sv_data, thresholds)
+    """
+
+    # Create copy
+    data = data.copy()
+
+    # Create mapping series for minimum and maximum Sv thresholds
+    freq_min_map = pd.Series({freq: vals["min"] for freq, vals in thresholds.items()})
+    freq_max_map = pd.Series({freq: vals["max"] for freq, vals in thresholds.items()})
+
+    # Apply frequency-specific thresholds
+    min_thresh = data["frequency"].map(freq_min_map)
+    max_thresh = data["frequency"].map(freq_max_map)
+
+    # Create mask
+    if "sv_mean" in data.columns:
+        mask = (data["sv_mean"] < min_thresh) | (data["sv_mean"] > max_thresh)
+    else:
+        raise KeyError(
+            "Could not apply Sv thresholds. Column 'sv_mean' could not be found within the "
+            "ingested acoustic DataFrame."
+        )
+
+    # Apply the thresholds
+    data.loc[mask, "sv_mean"] = -999.0
+
+    return data
+
+
+def sv_to_nasc(sv_linear, thickness_mean):
+    """
+    Convert volume backscattering coefficient (sv, linear) to NASC (sA).
+
+    This function converts linear volume backscattering coefficient values
+    to Nautical Area Scattering Coefficient using standard fisheries
+    acoustics equations. NASC is the integrated backscatter over a layer
+    expressed in units convenient for fisheries assessment.
+
+    Parameters
+    ----------
+    sv_linear : array-like
+        Volume backscattering coefficient in linear units (m^-1)
+    thickness_mean : array-like
+        Mean thickness of integration layer (m)
+
+    Returns
+    -------
+    nasc : array-like
+        Nautical area scattering coefficient (m^2 nmi^-2)
+
+    Notes
+    -----
+    The conversion follows standard fisheries acoustics equations:
+
+    .. math::
+        s_A = 4\\pi (1852)^2 \\int s_v \\, dz
+
+    where s_A is NASC, s_v is volume backscattering coefficient, and the
+    integral is approximated as s_v × layer_thickness. The factor 4π
+    accounts for spherical spreading, and (1852)² converts from m² to nmi².
+
+    References
+    ----------
+    .. [1] Simmonds, J. and MacLennan, D. (2005). Fisheries Acoustics:
+           Theory and Practice, 2nd ed. Oxford: Blackwell Science.
+    """
+    # From the equations in the image:
+    # sa = ∫ sv dz  (area backscattering coefficient)
+    # sA = 4π (1852)² sa  (NASC conversion)
+
+    # Constants
+    STERADIANS_SPHERE = 4 * np.pi  # 4π steradians
+    METERS_PER_NMILE_SQUARED = 1852**2  # (1852 m/nmi)²
+
+    # Calculate area backscattering coefficient (sa) by integrating sv over thickness
+    sa = sv_linear * thickness_mean  # ∫ sv dz approximated as sv × thickness
+
+    # Convert to NASC using sA = 4π(1852)² × sa
+    nasc = STERADIANS_SPHERE * METERS_PER_NMILE_SQUARED * sa
+
+    return nasc
+
+
+def aggregate_cells(data):
```

Comment:
If I read this function correctly, "aggregate" is not what this function is doing. The function "organizes" the  Sv and NASC data into different entries of a pivot table.

I suggest changing the function to avoid confusion. 

I also added #398 as a reminder to upgrade to use xarray as the data container.